### ISSUE TITLE
/teleport & /tp Add support for coordinates without a target

### ISSUE THREAD

**Issue by Allstreamer** [the-h-team/myEssentials#18]
Add Ability to /teleport or /tp to a location without target

**Comment by ms5984** [comment]
I'll finish the tab completion in a bit, but execution logic should be ready to test

**Comment by Allstreamer** [comment]
Relative coordinates like:
`/tp ~-50 ~20 ~10`

**Comment by Allstreamer** [comment]
would be a good addition

**Comment by ms5984** [comment]
> Relative coordinates like:
> `/tp ~-50 ~20 ~10`

Yeah, maybe we should delegate to /minecraft:teleport
The completion is richer, if we're going for the exact style I don't see the point in reinventing the wheel

@Hempfest Can you show me how to use the Luck-style tab completer code?

**Comment by Hempfest** [comment]
> > Relative coordinates like:
> > `/tp ~-50 ~20 ~10`
> 
> Yeah, maybe we should delegate to /minecraft:teleport
> The completion is richer, if we're going for the exact style I don't see the point in reinventing the wheel
> 
> @Hempfest Can you show me how to use the Luck-style tab completer code?

Are you talking about Brigadier tab completion?

**Comment by ms5984** [comment]
Yes

**Comment by ms5984** [comment]
^^just a rebase, force-push, it's the same commit from up top

**Comment by ms5984** [comment]
Basic tab completion is done. The relative coordinates should be doable, but I've got to plan that out a bit. Btw, here was the cheat sheet I made for myself so I could wrap my head around the syntax:
```
/teleport teleporting target
/teleport teleporting x y z
/teleport x y z
/teleport target
```
_so like, for instance in the second form, coords are relative to sender or relative to teleporting player? (check out how i've done the tab complete, i've already got some fallback-style stuff going on_
